lib/modules/manager/maven-wrapper/readme.md

```diff
@@ -0,0 +1,2 @@
+Configuration for Maven Wrapper updates.
+Changes here affect how Renovate updates the version of Maven in the wrapper, not how it uses the wrapper.
```

Comment:
From reading the docs, it's not clear why our users should care about any of this? It feels we're missing an explanation of the _why_ behind this maven-wrapper thing. 😉

I was expecting something like this:

> By default, Renovate updates Maven in the wrapper like [explain update method].
> This means that [insert problem with current method].
>
> To get around these problem(s), we have a Maven Wrapper manager, which does:
> 
> 1. text
> 2. text
> 3. text

Reply:
It's modelled on https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gradle-wrapper/readme.md